glib/src/object.rs

```diff
@@ -1173,8 +1167,8 @@ pub trait ObjectExt: ObjectType {
     fn is<T: StaticType>(&self) -> bool;
 
     fn get_type(&self) -> Type;
```

Comment:
Elsewhere we call this `fn type_(&self) -> ...`

Reply:
Yeah, I agree, this is the kind of stuff for which I wanted a general consensus. I added [this remark](https://github.com/fengalin/fix-getters/blob/main/rules/README.md#general-rules) in the `fix-getters-rules` README:

> [...] other solutions would be to generate `mut_`, `loop_`, `type_` or `r#mut`, `r#loop`, `r#type`, or even a mix depending on the keyword. E.g. we could keep `get_mut`, but use the quite common `type_` and `r#async` variantes. This is something I would like to discuss.

I'll update this in `fix-getters-rules` (and thus in Gir) in one go with others suggestions from other participants.

Reply:
`get_mut()` seems fine, the others should probably get a `_` appended. @GuillaumeGomez ?

Reply:
Note: full list of Reserved keyword for which we could debate ;)

https://github.com/fengalin/fix-getters/blob/ab8a8cceaf45b5ba98f8030541100437b1ecdb0f/rules/src/function.rs#L16

Reply:
`get_mut()` seems to be the only special case here, and there's `std::sync::Arc::get_mut()` so...

Reply:
I think `get_optional` and `get_some` make sense too.

Reply:
True

Reply:
I double checked. The keywords in the list can't be made into a call site name. E.g., you can't call `something.loop(...)`. So, beside the cases we mentioned previously, the keyword list above should be reviewed for candidates of alternatives such as `type_` or `r#async`.

Reply:
@GuillaumeGomez: what's your opinion? Which keywords should we substitute with which convention? So far, we agreed on:

- `get_mut`, `get_option` & `get_some`: unchanged.
- `get_type` -> `type_`.

Full list available from [here](https://github.com/fengalin/fix-getters/blob/ab8a8cceaf45b5ba98f8030541100437b1ecdb0f/rules/src/function.rs#L16). Most of them could be kept unchanged, but maybe we want more substitutes:

- `get_async` -> `r#async`?
- `loop` -> `loop_`?
- ...

Reply:
I'd go with adding `_` at the end. Simpler for users to be able to write it in their code.

Reply:
Done

Reply:
there's also `get_gtype` that I think should be renamed to `type_` instead of `gtype`

Reply:
Thanks. Will check this tomorrow. Too tired...

Reply:
@sdroege: how do you feel about `ParamSpec::new_gtype`? Is it ok to rename it to `ParamSpec::new_type`?
That's the only candidate for a `gtype` substitution I could find in `gtk-rs/gtk-rs`.

Reply:
No strong opinion either way :)

Reply:
Is it helpful to the user knowing it is a `GType`?

Reply:
Well, it's a `glib::Type`, not a `glib::GType`

Reply:
Good! switching to `ParamSpec::new_type` then. Thanks!